Task3/style/styles.css

```diff
@@ -0,0 +1,10 @@
+.circle{
+    background: linear-gradient(red,#e635ce,blue,rgb(28, 231, 238),rgb(15, 214, 15), yellow,red);
+    border-radius: 50%;
+    width: 250px;
+    height: 250px;
+    position: fixed; 
+    top: 50%;
+    left: 50%;
```

Comment:
what you did is right but need you to center the circle by using flex

Reply:
https://github.com/RahafSalah/FrontEndTraining/pull/5